frontend/src/pages/Courses/Courses.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {useState, useEffect} from 'react';
+import SideBar from "../../components/SideBar/SideBar.jsx";
+import styles from './CoursesStyles.module.css';
+
+const Courses = () => {
+    const apiUrl = import.meta.env.VITE_API_BASE_URL;
+    const [courses, setCourses] = useState([]);
+
+    useEffect(() => {
+        fetch(`${apiUrl}/course`,
+            {
+                method: 'GET',
+                headers: {
+                    'Content-Type': 'application/json',
+                    'Accept': 'application/json',
+                    'Authorization': `Bearer ${localStorage.getItem('token')}`
+                }
+            }
+            ).then(response => {
+            return response.json();
+        }).then(data => {
+            setCourses(data);
+        }).catch(error => {
+            console.log(error);
+        });
+    }, []);
+
+    return (
+        <div className={styles.courseContainer}>
+            <SideBar />
+            <ul className={styles.courseItems}>
+                {
+                    courses.map((course, index) => {
+                        return <li key={index}>{course.courseName}</li>
```

Comment:
It's bad practice to use `index` as a key in a react map function - this can cause state and performance issues. Use the course id instead.